### DISCUSSION TITLE
Upload a backup file using adb

### DISCUSSION THREAD

**Discussion by MobNiko** [LSPosed/LSPosed#2027] [Q&A]
Hi. 

I'm trying to load a backup file for LSPosed with module and app settings using the item in the "Restore" menu. 
I am using the adb command :  adb shell "su -c 'input tap 597 802'" , to generate a tap on the backup file "LSPosed_2022-07-08T15_43_16.686.lsp"
But it doesn't work. 
Is there any other way to restore all LSPosed settings using adb? Is there any way to restore backup using commands rather than using manual restore ?  

Thanks for the help!

**Comment by mywalkb** [comment]
The actual version of LSPosed has not CLI interface, but in my mod LSPosed_mod there is CLI interface, it still does not support backup/restore but thanks to your idea my mod will support it ASAP.

**Reply by MobNiko** [reply]
Thanks for your reply, will look forward to your release with the mod.

yes, i have spent a few days searching for different options, but they all don't work.
